amgut/handlers/new_participant.py

```diff
@@ -56,42 +51,16 @@ def post(self):
             self.redirect(media_locale['SITEBASE'] + "/authed/portal/?errmsg=%s" % errmsg)
             return
 
-        if is_juvenile == 'off' and is_exception:
-            errmsg = url_escape(tl["JUVENILE_CONSENT_EXPECTED"] %
-                                participant_name)
-            self.redirect(media_locale['SITEBASE'] + "/authed/portal/?errmsg=%s" % errmsg)
-            return
-
-        if is_juvenile == 'on':
-            # If they aren't already an exception, we need to verify them
-            if not is_exception:
-                alert_message = tl['MINOR_PARENTAL_BODY']
-
-                subject = ("AGJUVENILE: %s (ag_login_id: %s) is a child"
-                           % (participant_name, ag_login_id))
-
-                message = MESSAGE_TEMPLATE % (participant_name,
-                                              parent_1_name, parent_2_name,
-                                              deceased_parent,
-                                              self.current_user, kit_email)
-
-                try:
-                    send_email(message, subject, sender=kit_email)
-                    alert_message = tl['MESSAGE_SENT']
-                except:
-                    alert_message = media_locale['EMAIL_ERROR']
-
-                self.redirect(media_locale['SITEBASE'] + "/authed/portal/?errmsg=%s" % alert_message)
-                return
-
         human_survey_id = binascii.hexlify(os.urandom(8))
 
         consent= {'participant_name': participant_name,
                   'participant_email': participant_email,
                   'parent_1_name': parent_1_name,
                   'parent_2_name': parent_2_name,
-                  'is_juvenile': True if is_juvenile == 'on' else False,
+                  'is_juvenile': True if age_range != '18' else False,
```

Comment:
this is misleading, possible to have the value for `age_range` coming in as `'18_or_older'`?